javalib/src/main/scala/java/io/FileInputStream.scala

```diff
@@ -5,7 +5,7 @@ import scalanative.unsigned._
 import scalanative.unsafe._
 import scalanative.libc._, stdlib._, stdio._, string._
 import scalanative.nio.fs.UnixException
-import scalanative.posix.{fcntl, unistd}, unistd._
+import scalanative.posix.unistd, unistd._
```

Comment:
Since you are using `unistd.close(...)` you shouldn't need to add the `unistd._` import unless it is used elsewhere in the files.